easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,49 @@
+#//////////////////////////////////////////////////////////////////////////////
```

Comment:
look for `extract_step`, then browse down from there

the log message could be improved, since it doesn't seem to mention `start_dir` or `start dir` at all...